composer.json

```diff
@@ -44,7 +44,7 @@
     },
     "require-dev": {
         "php-parallel-lint/php-parallel-lint": "^1.2.0",
-        "slevomat/coding-standard": "^4.0.0 || ^6.4",
+        "slevomat/coding-standard": "^6.4",
```

Comment:
Please let's be as specific as possible here so we are sure to have the latest bug fixes: `^6.4.1`

(We don't need to to care about incompatibilities when Emogrifier is used as a library as this is a dev dependency.)

Reply:
I originally did have that in the `composer.json` file I failed to commit.  This time I tried to be more consistent with the change you committed in its place - but I see that was taken from the [change I committed](https://github.com/MyIntervals/emogrifier/pull/926/commits/32ddd12c352f5b36f24bd407731fd79d057f2659) to #926 which was in its infancy at the time.